### ISSUE TITLE
Add Chocolatey packages

### ISSUE THREAD

**Issue by daveaglick** [daveaglick/discoverdotnet#23]
Once multiple NuGet packages are implemented (#15), consider adding Choco packages to the mix

**Comment by gep13** [comment]
I was just coming here to add the Chocolatey Project into the mix, see this [PR](https://github.com/daveaglick/discoverdotnet/pull/89), and I was hoping to add a link to the Chocolatey Package itself, similar to what is done with NuGet packages.  Is this still something that you are thinking about?  Cheers

**Comment by daveaglick** [comment]
Yep, absolutely. Multiple packages + Choco support is probably the next big feature I'll add.
